app/js/controllers/angel.js

```diff
@@ -0,0 +1,253 @@
+    {
+      code : code,
+      grant_type:'authorization_code'
+    }}).then(function(response) {
+      //$location.path('/startups');
+  }, function(response) {
+
+      //$location.path('/startups');
+    // called asynchronously if an error occurs
+    // or server returns response with an error status.
+  });
+  $scope.curr_page=0;
+  $scope.lastpage;
+ 
+
+    var url="https://api.angel.co/1/startups?filter=raising&access_token=8cd27d851f1e4b69e27a29cc9606822c118586d4c8b168b5&callback=JSON_CALLBACK";
+    console.log(url);
+    $http.jsonp(url).
+    then(function(response) {
+      $scope.curr_page=response.data.page;
+      $scope.lastpage=response.data.last_page;
+      getStartups($scope.lastpage);
+      //console.log($rootScope.root.startups);
+    
+  }, function(response) {
+    
+    
+    console.log(response);
+    // called asynchronously if an error occurs
+    // or server returns response with an error status.
+  });
+
+  function getStartups(lastpage)
+  {
+    var i=1;
+    for(i=1;i<=15;i++)
+    {
+      $scope.startups.length=0; 
+      startupslocation.length=0;
+      
+      var url="https://api.angel.co/1/startups?filter=raising&access_token=8cd27d851f1e4b69e27a29cc9606822c118586d4c8b168b5&callback=JSON_CALLBACK&page="+i;
+      $http.jsonp(url).
+    then(function(response) {
+      
+      response.data.startups.forEach(function(ele){
+        
+        if(ele.locations[0])
+        {
+          
+        $scope.startups.push(
+        {
+          "startup_name" : ele.name,
+          "startup_url"  : ele.company_url,
+          "logo_url"     : ele.logo_url,
+          "product_desc" :ele.high_concept,
+          "location"     : ele.locations[0].display_name,
+          "quality"      : ele.quality
+          
+          });
+        if(ele.locations[0].display_name)
+        {
+         // console.log(ele.locations[0].display_name)
+        }
+        else
+        {
+          console.log("This startup has no locations");
+        }
+        
+        startupslocation.push(ele.locations[0].display_name);
+      }
+      
+      })
+      Geocode();
+
+        }, function(response) {
+    
+   
+    // called asynchronously if an error occurs
+    // or server returns response with an error status.
+  });
+
+
+    }
+   
+    
+    function Geocode()
+            { console.log("called");
+                
+                var locarray = {
+
+                    "places" : startupslocation
+                }
+               
+                
+            $http.jsonp('http://localhost:9000/api/geocode.json?callback=JSON_CALLBACK&minified=true', {params : { data : locarray } })
+            .success(function(data, status, headers, config) {
+              
+              var startupsMarker = {
+                "type": "FeatureCollection",
+                "features": []
+            };
+                //console.log( followers.propic[i]);
+                $scope.startups_status=startupslocation.length;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'startups_status' is not in camel case.](https://www.codacy.com/app/sudheesh1995/loklak_webclient/file/2550181458/issues/source?bid=3288768&fileBranchId=3288808#l163)